docs/user-guide.md

```diff
@@ -2,14 +2,16 @@
 
 Starting with descheduler release v0.10.0 container images are available in the official k8s container registry.
 
-Descheduler Version | Container Image                                     | Architectures           |
-------------------- |-----------------------------------------------------|-------------------------|
-v0.22.0             | k8s.gcr.io/descheduler/descheduler:v0.22.0          | AMD64<br>ARM64<br>ARMv7 |
-v0.21.0             | k8s.gcr.io/descheduler/descheduler:v0.21.0          | AMD64<br>ARM64<br>ARMv7 |
-v0.20.0             | k8s.gcr.io/descheduler/descheduler:v0.20.0          | AMD64<br>ARM64          |
-v0.19.0             | k8s.gcr.io/descheduler/descheduler:v0.19.0          | AMD64                   |
-v0.18.0             | k8s.gcr.io/descheduler/descheduler:v0.18.0          | AMD64                   |
-v0.10.0             | k8s.gcr.io/descheduler/descheduler:v0.10.0          | AMD64                   |
+Descheduler Version | Container Image                            | Architectures           |
+------------------- |--------------------------------------------|-------------------------|
+v0.23.1             | k8s.gcr.io/descheduler/descheduler:v0.23.1 | AMD64<br>ARM64<br>ARMv7 |
```

Comment:
Please update Descheduler version and Container Image to `v0.23.0`

Reply:
yes. done :) sorry for my mistake